### DISCUSSION TITLE
is this SDK compatible with php-7.2.6-apache??

### DISCUSSION THREAD

**Discussion by Isanfurg** [SkyAPM/SkyAPM-php-sdk#472] [Q&A] [answered]
I was trying to install the agent in a project but don't send anything to the oap server.

**Comment by heyanlong** [comment]
Need  to provide some information.

**Reply by Isanfurg** [reply]
> Need to provide some information.

I'm using PHP-7.2.6-Apache-Buster with lumen 5.7 and laravel 5.7. I'm just getting the instances on the oap-server (8.9.1).

**Comment by Isanfurg** [comment, marked as answer]
Finally I use Laravel-s connected to apache2 and works.
